# Message header
Header header

string[]  label
int8[]    type
float64[] min_distance
float64[] max_distance
float64[] mean_distance
float64[] xmin
float64[] ymin
float64[] xmax
float64[] ymax